README.md

```diff
@@ -0,0 +1,18 @@
+# FC calibration tool
+
+This is a tool that helps calibrate INAV FCs.
+
+## Installation
+
+To install and run this tool, you need to clone this repository and all its submodules:
+
+```
+$ git clone --recursive https://github.com/shellixyz/fc_calibration_tool.git
+$ sudo gem install serialport crc
+$ cd fc_calibration_tool
+$ ./calibrate <your FC's port>
+```
+
+Keep in mind that you need to connect to the FC **without connecting power**. You can
```

Comment:
Would be nice to specify power through USB and why it is not wise to do it: it will influence the readinds of the current sensor